### PR TITLE
Added Web Design & Development Toolkit

### PR DESCRIPTION
**Pull request by himanshuchandola**
Added Web Design & Development Toolkit in Others Section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ To add a resource or contribute, please check out our contribution guide [here](
   - [Open Source](#open-source)
   - [Standard Template Library](#standard-template-library)
   - [Programming Courses Free Online](#programming-courses-free-online)
+  - [Web Design & Development Toolkit](#web-design--development-toolkit)
```
Comment (shahednasser) [inline README.md:62]: Instead of creating a new topic for this resource, please add it under Design which is under Other

**Review by shahednasser — CHANGES_REQUESTED** [review]
Hi @himanshuchandola thank you for your contribution. I've added a comment to make changes below.

Also, please remove all changes in Gemfile and Gemfile.lock

**Comment by himanshuchandola** [comment]
I have updated Gemfile with your's repo one. and Moved Web Design and Development Kit Under Design

**Comment by shahednasser** [comment]
Great thanks!

**Pull request merged**